index.js

```diff
@@ -32,8 +33,7 @@ FileStore.prototype.get = function get(key, fn) {
   var self = this;
   var val = null;
   var data = null;
-  var fileKey = key.replace(':', '_');
-  var cacheFile = Path.join(self.tmpDir, fileKey + '.json');
+  var cacheFile = Path.join(self.tmpDir, sanitize(key) + '.json');
```

Comment:
Maybe set the key to `key = sanitize(key)` so you don't have to keep doing the function call? It would also look easier to read.